pwiz_tools/Skyline/SettingsUI/ViewLibraryDlg.cs

```diff
@@ -133,6 +137,13 @@ private string SelectedLibraryName
 
         public int PeptideDisplayCount { get { return listPeptide.Items.Count; } }
 
+        private MSGraphControl GraphControl
+        {
+            get { return msGraphExtension1.Graph; }

```

Comment:
Control naming could be better. Looks like you are accepting Visual Studio default with the name of the class and the 1 at the end.